### PR TITLE
Open51

### PR DESCRIPTION
**Pull request by shivamndave**
Summary of Changes:

* Addressed issue #51 
* If located within object that is removed, user is navigated to parent
* Also if object in any level above current object is removed, user is navigated to parent of removed object
* Adjusted RemoveActionSpec unit tests for this change

Changes address original issue? Y
Unit tests included and/or updated with changes? Y
Command line build passes? Y
Expect to pass code review? Y
Project-specific information isolated to appropriate branches? Y

### CODE REVIEW

File: platform/commonUI/edit/src/actions/RemoveAction.js [hunk 1]
```diff
@@ -80,21 +82,44 @@ define(
...
+            // Checks current object and ascendants of current
+            // object with object being removed, if the current
+            // object or any in the current object's path is being removed,
+            // navigate back to parent of removed object. 
+            function checkObjectNavigation(object, parentObject) {
+                // Traverse object starts at current location
+                var traverseObject = (navigationService).getNavigation();
+                
+                // Stop at ROOT of folder path
+                while (traverseObject.getId() !== ROOT_ID) {
```
Comment (VWoeltjen) [inline platform/commonUI/edit/src/actions/RemoveAction.js]: We could remove the need to explicitly look for the root ID by just checking if traverseObject is defined (or even just truthy), e.g. `while (traveseObject) {`

File: platform/persistence/elastic/bundle.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
             },
             {
                 "key": "ELASTIC_ROOT",
-                "value": "http://localhost:9200",
+                "value": "/elastic",
                 "priority": "fallback"
```
Comment (VWoeltjen) [inline platform/persistence/elastic/bundle.json]: This seems like an unrelated change - is there a reason this was included, or did it just tag along with another commit?

**Comment by VWoeltjen** [comment]
Looks good. Nice catch handling the case where the object being removed is a parent/ancestor of the navigated object, that would have been easy to miss.

Added a couple of comments inline, once these are addressed I think this is ready to merge.

File: platform/commonUI/edit/src/actions/RemoveAction.js (hunk 1 above)
Comment (shivamndave) [inline platform/commonUI/edit/src/actions/RemoveAction.js]: Sounds good, this also avoids any possibility for an infinite loop.

**Comment by shivamndave** [comment]
Regarding the comments:
* Change made to check the traverseObject instead of getting the id and
  comparing with ROOT, also adjusted unit tests for this
* The elastic bundle.json change was mixed in from a previous commit, it has been removed

**Comment by VWoeltjen** [comment]
1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
